aderyn/src/main.rs

```diff
@@ -109,6 +113,16 @@ fn main() {
         return;
     }
 
+    if cmd_args.root == "init" {
+        create_aderyn_toml_file_at(".".to_string());
+        return;
+    }
+
+    if cmd_args.init {
+        create_aderyn_toml_file_at(cmd_args.root);
+        return;
+    }
+
```

Comment:
I want to support two cases

* aderyn init 

(usually the first argument refers to the project root.  but if it's `init` then we treat it as a special case)

* aderyn optional/path/to/root --init

(expected way to initialize)

Doing this allows us to show users there's an option to init. in the help section. Otherwise if we only have the first case, there's no way to show that

Reply:
TRY THIS

* aderyn init <optional_path_here>

Reply:
I'll wait on this being implemented ^

Reply:
Sorry @alexroan I wasn't done yet, but I'll create an issue to get this done